pages/draft/2019-09/release-notes.md

```diff
@@ -1,172 +1,169 @@
----
-title: 2019-09 Release Notes
-section: docs
----
+### Draft 2019-09 Release Note
 
 For the vast majority of schema authors, we hope that these changes are minimally disruptive.
 
-The most likely to be frustrating is that `format` is no longer treated as a validation assertion _by default_ (although it is still possible for an application or user to configure a validator to treat it as one).  We decided this was acceptable because many schema authors are already extremely frustrated by its inconsistent behavior.
+The most likely to be frustrating is that `format` is no longer treated as a validation assertion *by default* (although it is still possible for an application or user to configure a validator to treat it as one). We decided this was acceptable because many schema authors are already extremely frustrated by its inconsistent behavior.
```

Comment:
todo: use underscores consistently to italicize.

Reply:
Yes, I tried having the underscore, but it goes back to the asterisk once the file is saved. 

![image](https://github.com/user-attachments/assets/5cec70e5-346b-4c75-94f0-06748d459d39)


Thought: It doesn't affect, as it produces the italicized result.